projects/js-toolkit/packages/liferay-cli/src/new/target-liferay/index.ts

```diff
@@ -0,0 +1,178 @@
+/**
+ * SPDX-FileCopyrightText: © 2017 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+ */
+
+import {
+	FilePath,
+	TemplateRenderer,
+	addConfigurationField,
+	addPkgJsonDependencies,
+	addPkgJsonScripts,
+	appendLines,
+	transformJsonFile,
+	transformTextFile,
+} from '@liferay/js-toolkit-core';
+import fs from 'fs';
+
+import * as facetConfiguration from '../facet-configuration';
+import * as facetLocalization from '../facet-localization';
+import * as facetPortlet from '../facet-portlet';
+import * as facetProject from '../facet-project';
+import prompt from '../util/prompt';
+
+import type {Options} from '..';
+
+export const name = 'Liferay Platform Project';
+
+export async function processOptions(options: Options): Promise<Options> {
+	options = await facetProject.processOptions(options);
+
+	options = await prompt(options, [
+		{
+			choices: ['portal-7.4-ga1'],
```

Comment:
nit: Would it make it easier to maintain if we imported all the possible choices from a config file or constant at the top of the file?

Reply:
Yes. I put that in-place because we only have one, but as soon as the second appeared, I would have moved it to a constant too :sweat_smile: